region/brinstar/blue/Blue Brinstar Energy Tank Room.json

```diff
@@ -30,23 +30,23 @@
     },
     {
       "id": 2,
-      "name": "Ceiling E-Tank Top Left Door (to Boulder Room)",
+      "name": "Top Left Door",
       "nodeType": "door",
       "nodeSubType": "blue",
       "nodeAddress": "0x0018ef2",
       "doorEnvironments": [{"physics": "air"}]
     },
     {
       "id": 3,
-      "name": "Energy Tank, Brinstar Ceiling",
+      "name": "Hidden Ceiling Item",
       "nodeType": "item",
       "nodeSubType": "hidden",
       "nodeItem": "ETank",
       "nodeAddress": "0x7879E"
     },
     {
       "id": 4,
-      "name": "Missile (blue Brinstar middle)",
+      "name": "Right Pedestal Item",
```

Comment:
Could just be "Pedestal Item" since there's only one pedestal item?

Reply:
I couldnt decide between `Right Item` and `Pedestal Item`. I think if I were to choose one it would be `Right Item` in this case.

Reply:
Sounds good